src/man/sssd.conf.5.xml

```diff
@@ -3433,11 +3433,6 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                             Default: <quote>id_provider</quote> is used if it
                             is set and can perform session related tasks.
                         </para>
-                        <para>
-                            <emphasis>NOTE:</emphasis> In order to have this feature
```

Comment:
> Hi,
> 
> although there is currently only Fleet Commander for `session_provider` I would mention it explicitly here in case other `session_provider`s are added in future which might work as SSSD user as well, so `.. In oder to have the Fleet Commander feature...`.

Why?
Those lines are being removed, because with this patch no privileges are required anymore for "Fleet Commander" to work.


> Additionally I think it would be good to have a section maybe in the `sssd` man page which explains how to run SSSD as "root" or unprivileged user and add a reference here.

I plan this on top of #6908

Reply:
> > Hi,
> > although there is currently only Fleet Commander for `session_provider` I would mention it explicitly here in case other `session_provider`s are added in future which might work as SSSD user as well, so `.. In oder to have the Fleet Commander feature...`.
> 
> Why? Those lines are being removed, because with this patch no privileges are required anymore for "Fleet Commander" to work.

ah, sorry, I was looking the other way round and thought this was added,

> 
> > Additionally I think it would be good to have a section maybe in the `sssd` man page which explains how to run SSSD as "root" or unprivileged user and add a reference here.
> 
> I plan this on top of #6908

Thanks.